### PR TITLE
Initial Apple Authentication node

### PR DESCRIPTION
**Pull request by r2m1**
Used the Apple pay node model to create this one that should be fine.
Full documentation on what is requested is available here: https://developers.ringcentral.com/engage/digital/guide/interactions/structured-messages/apple-auth

### CODE REVIEW

File: extensions/ringcentral-engage/src/nodes/sendAppleAuthenticate.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            key: "attachment",
+            label: "Attachment",
+            defaultCollapsed: true,
+            fields: [
+                "attachmentId"
+            ]
+        }
+    ],
+	form: [
+        { type: "field", key: "Text" }
```
Comment (AlexTeusz) [inline extensions/ringcentral-engage/src/nodes/sendAppleAuthenticate.ts:41]: I think you mean  `response_body` should be the `key` in this line.

**Review by AlexTeusz — COMMENTED** [review]
Please change the thing a requested :) then it is ready

File: extensions/ringcentral-engage/src/nodes/sendAppleAuthenticate.ts (hunk 1 above)
Comment (AlexTeusz) [reply extensions/ringcentral-engage/src/nodes/sendAppleAuthenticate.ts:41]: @r2m1 could you add the changes please? Thank you very much :)

Comment (r2m1) [reply extensions/ringcentral-engage/src/nodes/sendAppleAuthenticate.ts]: Should be ok now !

**Pull request merged**